src/main/java/training/endava/playground/generics/MainApp.java

```diff
@@ -0,0 +1,93 @@
+package training.endava.playground.generics;
+
+import training.endava.playground.generics.types.Article;
+
+import java.util.*;
+
+public class MainApp {
+
+    public static void main(String[] args) {
+
+        List<Article> articles=new ArrayList<>();
+
+        ArrayList<String> tagA=new ArrayList<String>();
```

Comment:
No need to specify the type (String) when creating the object. As best practice, you should use List for type declaration, if you are not constraint.